pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -93,6 +95,7 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	// Any state needed by handler functions
 	ctx := zedmanagerContext{
 		globalConfig: types.DefaultConfigItemValueMap(),
+		hvTypeKube:   base.IsHVTypeKube(),
```

Comment:
Why is it necessary to have a flag on the `zedManagerContext` to indicate that the hv is kube? Is it consumed solely at [this line](https://github.com/lf-edge/eve/pull/3833/files#diff-de84b12c2ce63e5d98657e2297135ce3af53221cbb4611fa7af9534c9e3a7e98R66) and [this line](https://github.com/lf-edge/eve/pull/3833/files#diff-de84b12c2ce63e5d98657e2297135ce3af53221cbb4611fa7af9534c9e3a7e98R84)?

If so, I understand the need, but wonder if we are adding more duplicated properties in places? 

I notice that `base` is a package, so `base.IsHVTypeKube()` is a global function, not a property? Is it evaluating the system as a whole?

Reply:
Because every micro service has there own context and good to have this flag to indicate its kubevirt eve. May be right now its used only at couple of places. In future when the clustering code comes up it will be helpful. 

base.IsHVTypeKube() merely reads a file /run/eve-hv-type.  We do not want to read from disk every time. Its cleaner to have a flag in the context.

Reply:
> base.IsHVTypeKube() merely reads a file /run/eve-hv-type. We do not want to read from disk every time. Its cleaner to have a flag in the context

OK, that makes sense.